force-app/main/default/lwc/lmsSubscriberWebComponent/__tests__/lmsSubscriberWebComponent.test.js

```diff
@@ -0,0 +1,171 @@
+import { createElement } from 'lwc';
+import LmsSubscriberWebComponent from 'c/lmsSubscriberWebComponent';
+import { ShowToastEventName } from 'lightning/platformShowToastEvent';
+import { getRecord } from 'lightning/uiRecordApi';
+import {
+    registerLdsTestWireAdapter,
+    registerTestWireAdapter
+} from '@salesforce/sfdx-lwc-jest';
+
+import { subscribe, MessageContext, publish } from 'lightning/messageService';
+import recordSelected from '@salesforce/messageChannel/Record_Selected__c';
```

Comment:
Use uppercase for the channel name because it's a constant (unlike the other imports). BTW this is consistent with doc.

Reply:
done.

Reply:
Did you forget that one?

Reply:
apparently.